terraform/environments/digital-prison-reporting/modules/ec2/scripts/bootstrap_apps.sh

```diff
@@ -218,8 +219,8 @@ sudo cat <<EOF > $bodmis_portforwarder_script
 
 unset KUBE_CONFIG; unset KUBECONFIG
 
-export KUBE_CONFIG=$kubeconfig
-export KUBECONFIG=$kubeconfig
+export KUBE_CONFIG=$bodmis_kubeconfig
```

Comment:
Both of these ENV variables are assigned to $bodmis_kubeconfig.
Was this intentional or on was meant to be assigned to nomis?

Reply:
Yes it is required, as some config is reading KUBE_CONFIg and kubectl is expecting KUVECONFIG variable